pkgs/top-level/all-packages.nix

```diff
@@ -18489,7 +18489,9 @@ with pkgs;
     libpng = libpng12;
   };
 
-  mnemosyne = callPackage ../games/mnemosyne { };
+  mnemosyne = libsForQt5.callPackage ../games/mnemosyne {
+    inherit (python3Packages) buildPythonApplication pyqt5 matplotlib cherrypy cheroot webob pillow;
```

Comment:
Please don't make these kind of changes. They can only cause more work when there are incompatible versions.